CMakeLists.txt

```diff
@@ -1055,6 +1055,16 @@ if(CPACK_GENERATOR MATCHES "NSIS")
   endif()
 endif()
 
+# CPack: Windows System w/ WiX
+if(CPACK_GENERATOR MATCHES "WIX")
+  set(CPACK_WIX_UPGRADE_GUID cb6825fd-37e6-4596-a55d-6d490d4fe178)
```

Comment:
place a comment indicating from where this unique id comes from or maybe reference https://cmake.org/cmake/help/v3.3/module/CPackWIX.html